libs/zzz/db/src/Database/DataEntries/DisplayWengineEntry.ts

```
@@ -0,0 +1,107 @@
+      speciality,
+      locked,
+      showEquipped,
+      showInventory,
+      locations,
+    }
+    return data
+  }
+  override set(
+    value:
+      | Partial<IDisplayWengine>
+      | ((v: IDisplayWengine) => Partial<IDisplayWengine> | void)
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)